_drafts/agile.md

```diff
@@ -15,96 +15,96 @@ keywords:
   - agile is evil
 ---
 
-[Agile Manifesto](http://www.agilemanifesto.org/) is nowadays a Bible of
-so many software teams. It contains
-[12 principles](http://agilemanifesto.org/principles.html), which are
-supposed to show us how software development must be organized.
-They were invented in 2001.
-I like them and agree with all of them. However, in practice, most software teams
-misunderstand them all. Here is my summary of what's going on and my
-own interpretation of each principle.
+Nowadays, [Agile Manifesto](http://www.agilemanifesto.org/) is a Bible of
+numerous software teams. It contains
+[12 principles](http://agilemanifesto.org/principles.html) which
+ show us how software development should be organized.
+These principles were invented in 2001.
+Generally, I like and agree with all of them. However, in practice, most software teams
+misunderstand them. Consequently, here is a summary of what's going on and my
+interpretation of each principle.
 
 <!--more-->
 
 **Principle #1**:
 "Our highest priority is to satisfy the customer through early and continuous delivery of valuable software."
 
-Staying focused on "satisfy the customer" Agile adepts totally forget about
+By focusing on "satisfy the customer", Agile adepts totally forgets about
 the "through" part. They think that a [happy customer]({% pst 2015/jan/2015-01-26-happy-boss-false-objective %})
 is their true objective, while "continuous delivery" is something that obviously
-helps, while is not really critical. It is quite the opposite &mdash; the customer
-will be satisfied **if** the software is created and delivered right. If the
+helps, though not crucially. However, this is quite the opposite &mdash; the customer
+will be satisfied **if** the software is perfectly created and delivered. If the
 customer is not satisfied, we find another customer &mdash; that's the
-true spirit a professional software team should adhere to. That's what
-the Manifesto means, I believe. We make sure that our process is
-"early and continuous" and the customer will be satisfied. We focus on improving
-our process, not on satisfying the customer. Satisfaction is the consequence, not
+true spirit a professional software team should adhere to. I believe that's what
+ the Manifesto means. We make sure that our process is
+"early and continuous", which will result to customer satisfaction. We focus on improving
+our process, not satisfying the customer. Satisfaction is the consequence, not
 the primary objective.
 
 {% quote If the customer is not satisfied, we find another customer %}
 
 **Principle #2**:
-"Welcome changing requirements, even late in development. Agile processes harness change for the customer's competitive advantage."
+"Welcome change requirements, even late in development. Agile processes harness change for the customer's competitive advantage."
```

Comment:
@gabbylisper we should not change that, this text comes from the Manifesto itself. I'm just quoting it here